lib/sycamore/sycamore/utils/aryn_config.py

```diff
@@ -0,0 +1,41 @@
+import os
+import pathlib
+import yaml
+import logging
+
+_DEFAULT_PATH = os.path.join(pathlib.Path.home(), ".aryn", "config.yaml")
+
+
+class ArynConfig:
+    def __init__(self, **kwargs):
+        self.__dict__.update(kwargs)
+
+
+def get_aryn_api_key(requested_yaml_config_path: str = "") -> str:
+    api_key = os.environ.get("ARYN_API_KEY")
+    if api_key:
+        return api_key
+
+    global_aryn_config = get_aryn_config(requested_yaml_config_path)
```

Comment:
It looks like we parse the config file every time and throw the result away right after.

Reply:
Fixed.